packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -299,6 +305,19 @@ export default class AgoraIpOrderCtrl {
 
   redirectToPaymentPage() {
     let productToOrder = null;
+
+    this.atInternet.trackClick({
+      name: `${TRACKING_PREFIX}confirm_${get(
+        this.model,
+        'selectedService.type',
+      )}_${get(this.model.params, 'selectedOffer.planCode')}_${get(
+        this.model.params,
+        'selectedQuantity',
+        1,
+      )}_${get(this.model.params, 'selectedCountry.code', null)}`,
+      type: 'action',
+    });
```

Comment:
May be we can improved like this:

```suggestion
    const { params } = this.model;
    const serviceType = params.selectedService.type;
    const offerPlanCode = params.selectedOffer.planCode;
    const quantity = params.selectedQuantity || 1;
    const countryCode = params.selectedCountry.code || null;

    this.atInternet.trackClick({
      name: `${TRACKING_PREFIX}confirm_${serviceType}_${offerPlanCode}_${quantity}_${countryCode}`,
      type: 'action',
    });
```